tests/operator_test.go

```diff
@@ -1236,6 +1236,8 @@ var _ = Describe("ALL Operator tests", func() {
 				serverSecretNames := []string{"cdi-apiserver-server-cert", "cdi-uploadproxy-server-cert"}
 
 				ts := time.Now()
+				// Time comparison here is in seconds, so make sure there is an interval
+				time.Sleep(time.Second)
```

Comment:
Nice!

Reply:
So this failed if it was too fast? < 1 second?

Reply:
yep